port/unix/omrsysinfo.c

```diff
@@ -146,23 +146,25 @@
 #endif
 #endif
 
+#include <unistd.h>
```

Comment:
Why does this need to move?

Reply:
Reverted it; it was an attempt to enforce order between **"**NAME**"** and **<**NAME**>** includes.